docs/src/style_recommendations.md

```diff
@@ -2,6 +2,10 @@
 
 In this page we compile the general recommendations in terms of best coding practices, conventions and style for developers contributing to the ODINN ecosystem.
 
+## SciML code style
+
+In the `ODINN.jl` ecosystem, we follow the [SciML code style](https://github.com/SciML/SciMLStyle). For now, it is enforced only during review, but we plan to soon deploy an automatic code formatter which will enforce that. 
```

Comment:
@albangossard I have added this section. I don't know the details and exact plans about the code formatter. Maybe you want to complement this.